src/main/java/org/deepsymmetry/beatlink/data/OpusProvider.java

```diff
@@ -51,98 +59,135 @@ public boolean isRunning() {
     }
 
     /**
-     * Holds the ZIP filesystem of the media export mounted for USB slot 1 of the Opus Quad.
+     * Check whether we currently have any attached media
+     *
+     * @return true if any database has attached data
      */
-    private final AtomicReference<FileSystem> usb1filesystem = new AtomicReference<>();
+    @API(status = API.Status.STABLE)
+    public boolean hasAttachedArchive() {
```

Comment:
We won’t need this method; nothing should care whether any metadata archives are attached.